src/passport-saml/saml.ts

```diff
@@ -131,6 +131,10 @@ class SAML {
       throw new Error("Invalid property: cert must not be empty");
     }
 
+    if (options.wantAssertionsSigned && !options.cert) {
+      throw new Error('"cert" config parameter is required for signed assertions');
```

Comment:
This error message looks a bit misleading. I suggest: "cert config parameter with IdP's certificate is required to validate assertion's signature".

side note: @cjbarth @markstos IMHO this if statement and the one at the line 130 of this pull request branch could be combined into single block which checks that cert configuration option is defined and that value of cert is not undefined (that check would cover also this particular blocks situation because it would make sure that cert is provided). Now these two if statements are used to keep ignoring missing cert option situation and result is issues 180 and 523